packages/firestore/test/util/helpers.ts

```diff
@@ -233,6 +239,38 @@ export function bound(
   return new Bound(components, before);
 }
 
+export function queryData(
+  targetId: TargetId,
+  queryPurpose: QueryPurpose,
+  path: string
+): QueryData {
+  return new QueryData(query(path)._query, targetId, queryPurpose);
+}
+
+export function docInsertRemoteEvent(
```

Comment:
Why "Insert" ?  Do we use that term elsewhere?  Should this be docAddedRemoteEvent() or something?